lib/omnibus/git_cache.rb

```diff
@@ -32,7 +32,7 @@ class GitCache
     # will not have the generated content, so these snapshots would be
     # incompatible with the current omnibus codebase. Incrementing the serial
     # number ensures these old shapshots will not be used in subsequent builds.
-    SERIAL_NUMBER = 1
+    SERIAL_NUMBER = 2
```

Comment:
There are a few reasons:

1. Cache might not contain the license info collected by `chef_berkshelf` plugin. Technically anytime we add a new dep mgr, we need to bump this. 
2. The name of the project we're passing has changed which requires us to invalidate the old caches.